OJP/OJP_Requests.xsd

```diff
@@ -203,7 +203,6 @@
 			<xs:extension base="AbstractOJPServiceRequestStructure">
 				<xs:sequence>
 					<xs:group ref="TripRefineRequestGroup"/>
-					<xs:element ref="siri:Extensions" minOccurs="0"/>
```

Comment:
This is an unrelated change. Must not be in this pull request.

Reply:
ok will remove it.

Reply:
... from this PR...